common/src/main/java/io/apicurio/registry/rest/v2/beans/SystemInfo.java

```diff
@@ -32,7 +32,7 @@
     private String description;
     @JsonProperty("version")
     private String version;
-    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss'Z'", timezone = "UTC")
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ssZ", timezone = "UTC")
```

Comment:
you forgot to revert this change

Reply:
Oh, but, is there a reason why everywhere else is yyyy-MM-dd'T'HH:mm:ssZ and only on this place there is yyyy-MM-dd'T'HH:mm:ss'Z'?